### DISCUSSION TITLE
Catch message about user join to group by invite

### DISCUSSION THREAD

**Discussion by devopg** [telegraf/telegraf#1779] [Q&A]
Hello. How i can catch message that a user join to group by invite ? 
in chat i see: 
"User777 joined the group at the invitation link"

I try 
`bot.on(message('new_chat_member'), async (ctx) => {})`
but it dont contained information about invite link

**Comment by MKRhere** [comment]
You want the `bot.on("chat_member", handler)` update. But chat_member is not sent by Telegram by default. To receive it, you must pass all update types you want to receive to `allowedUpdates`:

```TS
bot.launch({
    allowedUpdates: [
      'update_id',
      'message',
      'edited_message',
      'channel_post',
      'edited_channel_post',
      'inline_query',
      'chosen_inline_result',
      'callback_query',
      'shipping_query',
      'pre_checkout_query',
      'poll',
      'poll_answer',
      'my_chat_member',
      'chat_member',
      'chat_join_request'
    ],
});
```
